src/utils/exerciseServer.ts

```diff
@@ -2,13 +2,18 @@
 
 import sh = require("@typescript/server-harness");
 import fs = require("fs");
+import type typescript = require("typescript/lib/tsserverlibrary");
```

Comment:
Why are we mixing ESM imports and `require`s here?

Reply:
Because the `require`s don't support destructuring 😭

Open to changing this in the future but I'll stick this this as-is for now